src/test/scala/com/databricks/spark/sql/perf/mllib/MLLibSuite.scala

```diff
@@ -1,10 +1,29 @@
 package com.databricks.spark.sql.perf.mllib
 
-import org.scalatest.FunSuite
+import org.scalatest.{BeforeAndAfterAll, FunSuite}
 
+import org.apache.spark.{SparkConf, SparkContext}
 import org.apache.spark.sql.Row
 
-class MLLibTest extends FunSuite {
+class MLLibSuite extends FunSuite with BeforeAndAfterAll {
+
+  private var sc: SparkContext = _
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    val sparkConf = new SparkConf().setAppName("MLlib QA").setMaster("local[2]")
+    sc = SparkContext.getOrCreate(sparkConf)
+  }
+
+  override def afterAll(): Unit = {
+    super.afterAll()
+    if (sc != null) {
+      sc.stop()
+    }
+    // To avoid RPC rebinding to the same port, since it doesn't unbind immediately on shutdown
```

Comment:
curious: Is this from somewhere or because of a problem you hit?

Reply:
I copied this from the `LocalSparkSession` trait, https://livegrep.dev.databricks.com/view/databricks/spark-master/core/src/test/scala/org/apache/spark/LocalSparkContext.scala#L55